src/fal/cli/dbt_runner.py

```diff
@@ -43,7 +46,7 @@ def logs(self) -> List[Dict[str, Any]]:
         return self._logs
 
 
-def dbt_run(args, models_list: List[str]):
+def dbt_run(args, models_list: List[str], target_path: str, run_index: int = 0):
```

Comment:
maybe we should NOT offer a default for it, to avoid bugs of overwriting old files.